pkg/service/schema.go

```diff
@@ -196,13 +194,41 @@ func (i IntPropertySchema) validate(context string, value interface{}) error {
 	if value == nil {
 		return nil
 	}
-	floatVal, ok := value.(float64)
-	if !ok {
-		return NewValidationError(context, "field value is not of type int64")
-	}
-	val := int64(floatVal)
-	if floatVal != float64(val) {
-		return NewValidationError(context, "field value is not of type int64")
+	var val int64
+	if floatVal, ok := value.(float64); ok {
+		val = int64(floatVal)
+		if floatVal != float64(val) {
+			return NewValidationError(context, "field value is not of type integer")
+		}
+	} else if floatVal, ok := value.(*float64); ok {
+		val = int64(*floatVal)
+		if *floatVal != float64(val) {
+			return NewValidationError(context, "field value is not of type integer")
+		}
+	} else if floatVal, ok := value.(float32); ok {
+		val = int64(floatVal)
+		if floatVal != float32(val) {
+			return NewValidationError(context, "field value is not of type integer")
+		}
+	} else if floatVal, ok := value.(*float32); ok {
+		val = int64(*floatVal)
+		if *floatVal != float32(val) {
+			return NewValidationError(context, "field value is not of type integer")
+		}
+	} else if intVal, ok := value.(int64); ok {
+		val = intVal
+	} else if intVal, ok := value.(*int64); ok {
+		val = *intVal
+	} else if intVal, ok := value.(int32); ok {
+		val = int64(intVal)
+	} else if intVal, ok := value.(*int32); ok {
+		val = int64(*intVal)
+	} else if intVal, ok := value.(int); ok {
+		val = int64(intVal)
+	} else if intVal, ok := value.(*int); ok {
+		val = int64(*intVal)
+	} else {
+		return NewValidationError(context, "field value is not of type integer")
```

Comment:
All of the above just makes validations more tolerant of the types in the map that _might_ represent an integer.